src/takeLastWhile.ts

```diff
@@ -0,0 +1,46 @@
+import { purry } from './purry';
+
+/**
+ * Returns elements from the end of the array until the predicate returns false.
```

Comment:
See my comment about the description below

Reply:
Done